apps/server/src/modules/user-login-migration/service/school-migration.service.spec.ts

```diff
@@ -225,25 +225,26 @@ describe('SchoolMigrationService', () => {
 		describe('when admin completes the migration', () => {
 			it('should call getSchoolById on schoolService', async () => {
 				const expectedSchoolId = 'expectedSchoolId';
+				const migrationStartedAt = new Date();
 				const users: Page<UserDO> = new Page([userDoFactory.buildWithId()], 1);
 				userService.findUsers.mockResolvedValue(users);
 
-				await service.completeMigration(expectedSchoolId);
+				await service.completeMigration(expectedSchoolId, migrationStartedAt);
 
 				expect(schoolService.getSchoolById).toHaveBeenCalledWith(expectedSchoolId);
 			});
 
 			it('should call findUsers on userService', async () => {
-				const { schoolId, oauthMigrationPossible } = setup();
+				const { schoolId, oauthMigrationStart } = setup();
 				const users: Page<UserDO> = new Page([userDoFactory.buildWithId()], 1);
 				userService.findUsers.mockResolvedValue(users);
 
-				await service.completeMigration(schoolId);
+				await service.completeMigration(schoolId, oauthMigrationStart);
 
 				expect(userService.findUsers).toHaveBeenCalledWith({
 					schoolId,
 					isOutdated: false,
-					lastLoginSystemChangeSmallerThan: expect.objectContaining<Date>(oauthMigrationPossible) as Date,
+					lastLoginSystemChangeSmallerThan: expect.objectContaining<Date>(oauthMigrationStart) as Date,
```

Comment:
With newer jest version it should be possible to remove "as Date".

Reply:
thanks for the hint :)